src/secure_random.cr

```diff
@@ -29,7 +69,15 @@ module SecureRandom
     slice
   end
 
-  def self.uuid
+  # Generates a UUID (Universally Unique Identifier)
+  # 
+  # It generates a random v4 UUID. Check [RFC 4122](https://tools.ietf.org/html/rfc4122#section-4.4)
```

Comment:
We really should explicitly mention Section 4.4, the link would be gone if anybody every does plaintext renderings of this.